Sources/SwiftParser/Nominals.swift

```diff
@@ -333,7 +333,7 @@ extension Parser {
   }
 
   mutating func parsePrimaryAssociatedTypes() -> RawPrimaryAssociatedTypeClauseSyntax {
-    let langle = self.consumeAnyToken(remapping: .leftAngle)
+    let langle = self.consumePrefix("<", as: .leftAngle)
```

Comment:
Did you search for `consumeAnyToken(.*leftAngle` and `rightAngle` to make sure we don't have any more of these 😅?

Reply:
Yes, all other `consumeAnyToken(remapping:)` calls are fine.